temp.py

```diff
@@ -1 +1,3 @@
 # This is Temp File
+
+# This is python file . pls chnag to js
```

Comment:
please keep it same, and add print function

Reply:
done